qtfred/ui/ShipCustomWarpDialog.ui

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>fso::fred::dialogs::ShipCustomWarpDialog</class>
+ <widget class="QDialog" name="fso::fred::dialogs::ShipCustomWarpDialog">
+  <property name="windowModality">
+   <enum>Qt::NonModal</enum>
+  </property>
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>333</width>
+    <height>356</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Edit Parameters</string>
+  </property>
+  <property name="modal">
+   <bool>true</bool>
+  </property>
+  <layout class="QVBoxLayout" name="layout">
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Note: All these parameters are optional.</string>
```

Comment:
I meant that the full usability text that is used in FRED should also be used in qtFRED:
> All of these parameters are optional, and a mission designer will usually not need to override more than one or two.  These parameters are the same as those found in ships.tbl.

Reply:
Not particularly happy with this one as the text seems rather verbose and unnecessary but don't feel strongly enough to fight you on this.